lib/web_ui/lib/src/engine/embedder.dart

```diff
@@ -216,8 +218,12 @@ class FlutterViewEmbedder {
         .instance.semanticsHelper
         .prepareAccessibilityPlaceholder();
 
+    final DomElement announcementsElement = createDomElement('div');
```

Comment:
Does it literally need to be `<div>`? For consistency with other top-level elements it would be good for it to have a custom name, such as `<flt-announcement-host>`.

Reply:
Sure, updated per your suggestion and also updated the two live elements and confirmed that things still work fine on ChromeVox on our app.